pkg/network/node/subnets.go

```diff
@@ -103,6 +103,10 @@ func (node *OsdnNode) handleDeleteHostSubnet(obj interface{}) {
 	if hs.HostIP == node.localIP {
 		return
 	}
+	if _, exists := node.hostSubnetMap[string(hs.UID)]; !exists {
```

Comment:
Can we still delete the rules for the subnet portion? Just skip the host part of the openflow?
I am not sure if those rules are really non-existent, while we clearly do not want the host part to be deleted of course.

Reply:
Agreed. I feel that this fix make things better. May be not ideal, but better than what we see today.